File: exercicios em C/q84.cpp
#include <stdio.h>


int main(void)
{
	int  likes, n, soma, i, media;
	printf ("\n\nEste programa determina a media de curtidas a qual voce recebe em um dia.");
	printf ("\n\nDigite a quantidade de postagens q voce faz no dia: ");
	scanf ("%d", &n);
	for (i = 1; i <= n; i++){
		printf ("\n\nDigite a quantidade de curtidas: ");
		scanf ("%d", &likes);
		soma = likes + soma;
	}
	media = soma / n;
	printf ("A media de like e: %d\n", media);
	printf("\n\nObrigado por utilizar nosso sistema!");
	return 0;
	
}

File: exercicios em Python/q84.py
import os

print("\n\nEste programa determina a media de curtidas a qual voce recebe em um dia.")
n = int(input("\n\nDigite a quantidade de postagens q voce faz no dia: "))
i = 1
soma = 0
media = 0
while i <= n:
        print("\n\nDigite a quantidade de curtidas da ",i," publicaca: ")
        likes = int(input(""))
        soma = int(likes + soma)
        i = i + 1
        
media = int(soma / n)
print("A media de like e: ",media)
print("\n\nObrigado por utilizar nosso sistema!")
os.system("pause")
